src/classes/ADDR_Account_TDTM.cls

```diff
@@ -48,123 +51,122 @@ public class ADDR_Account_TDTM extends TDTM_Runnable {
-           !ADDR_Addresses_TDTM.equalsCaseSensitive(acc1.BillingState, acc2.BillingState) ||
-           !ADDR_Addresses_TDTM.equalsCaseSensitive(acc1.BillingPostalCode, acc2.BillingPostalCode) ||
-           !ADDR_Addresses_TDTM.equalsCaseSensitive(acc1.BillingCountry, acc2.BillingCountry);
-
-        if (!isChanged && ADDR_Addresses_TDTM.isStateCountryPicklistsEnabled) {
-            isChanged = 
-                !ADDR_Addresses_TDTM.equalsCaseSensitive(string.valueOf(acc1.get('BillingStateCode')), string.valueOf(acc2.get('BillingStateCode'))) || 
-                !ADDR_Addresses_TDTM.equalsCaseSensitive(string.valueOf(acc1.get('BillingCountryCode')), string.valueOf(acc2.get('BillingCountryCode')));
-        }
-        UTIL_Debug.debug('****Address changed: ' + isChanged);
-        return isChanged;   
-
+        hasRunAddrTrigger = true;
+                            
+        return dmlWrapper;    
     }
-
+    
     /*******************************************************************************************************
     * @description for each Account, create a new default address and add it to dmlWrapper
-    * @param listAcc a list of Accounts
+    * @param ListAcc a List of Accounts
     * @param dmlWrapper to hold the Addresses that need creating
     * @return void
     ********************************************************************************************************/
-    private void createAddrFromAcc(list<Account> listAcc, DmlWrapper dmlWrapper) {
-        list<Address__c> listAddr = new list<Address__c>();
-        for (Account acc : listAcc) {
+    private void createAddrFromAcc(List<Account> ListAcc, DmlWrapper dmlWrapper) {
+        List<Address__c> ListAddr = new List<Address__c>();
+        for (Account acc : ListAcc) {
             Address__c addr = new Address__c();
             addr.Parent_Account__c = acc.Id;
             addr.Default_Address__c = true;
             addr.Latest_Start_Date__c = system.today();
             addr.Latest_End_Date__c = null;
-            ADDR_Addresses_TDTM.copyAddressStdSObjAddr(acc, 'Billing', addr, null);            
-            listAddr.add(addr);         
+            ADDR_Addresses_UTIL.copyAddressStdSObjAddr(acc, 'Billing', addr, null);            
+            ListAddr.add(addr);
+            UTIL_Debug.debug('****New address record: ' + JSON.serializePretty(addr));
         }
-        // since coming from an account address, there is no Address Type, so exclude it from the match testing.
-        ADDR_Addresses_TDTM.createNonDuplicateAddresses(listAddr, dmlWrapper, false);
+        //since coming from an account address, there is no Address Type, so exclude it from the match testing.
+        List<Address__c> newAddrs = ADDR_Addresses_UTIL.createNonDuplicateAddresses(ListAddr, dmlWrapper, false);
+        
+        //Match each Account with each new address
+        List<ID> accIDsToUpdate = new List<ID>();
```

Comment:
Variables should use the `lowerCamelCase` naming convention. Please rename accordingly.